app/assets/stylesheets/application.scss

```diff
@@ -14,4 +14,5 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
 @import "lists";
+@import "gift_requests";
```

Comment:
It should be named something other than `gift_requests` if it applies to more than just the gift requests views.

You can still nix the lists import, I think?